### ISSUE TITLE
Why are half of DowncastHelpers protected?

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-engine#1729]
I need `insertContent()` in my plugin but it's protected.

![image](https://user-images.githubusercontent.com/156149/56409754-b8f35e00-627a-11e9-8ed2-4b1ba4771c2e.png)

**Comment by Reinmar** [comment]
cc @pjasiun @jodator

**Comment by jodator** [comment]
It might be that some methods were protected before (`wrap()` maybe) but it looks like there's no need for that. Also it might be some typo :(

**Comment by scofalik** [comment]
AFAIR it was decided to hide those methods (at least those which are not used) so that users will not be confused which helper to use.

**Comment by pjasiun** [comment]
> AFAIR it was decided to hide those methods (at least those which are not used) so that users will not be confused which helper to use.

Yep. We wanted to simplify the API (which is already complicated) so exposed only the part we know that need to be exposed. However, I am sure that we may expose some more utils if needed.
